### PR TITLE
v8: update jQuery and jQuery UI

### PR DESCRIPTION
**Pull request by iOvergaard**
### Prerequisites

- [x] I have added steps to test this contribution in the description below

### Description

This updates jQuery to 3.6.0 and jquery-ui to 1.14.1 because of a few vulnerabilities as [described by Snyk](https://snyk.io/vuln/npm:jquery-ui@1.12.1).

Reference: CVE-2021-41184

### CODE REVIEW

**Comment by Zeegaan** [comment]
Tests out just fine, LGTM 👍

File: src/Umbraco.Web.UI.Client/package.json [hunk 1]
```diff
@@ -35,8 +35,8 @@
     "diff": "3.5.0",
     "flatpickr": "4.6.9",
     "font-awesome": "4.7.0",
-    "jquery": "^3.6.0",
-    "jquery-ui-dist": "1.12.1",
+    "jquery": "3.6.0",
+    "jquery-ui-dist": "1.13.1",
```
Comment (RyuLindow) [inline src/Umbraco.Web.UI.Client/package.json:39]: was supposed to be 1.14.1 :)

Comment (RyuLindow) [reply src/Umbraco.Web.UI.Client/package.json:39]: was supposed to be 1.14.1 @iOvergaard :)

Comment (frederiktoft) [reply src/Umbraco.Web.UI.Client/package.json:39]: 1.13.2 to be precise.

Comment (iOvergaard) [reply src/Umbraco.Web.UI.Client/package.json:39]: @frederiktoft Sorry but 1.13.2 did not exist on June 10th, 2022 when this pull request was made. It wasn't released until a [month later](https://github.com/jquery/jquery-ui/releases/tag/1.13.2) and only landed on npm in this distributed version a [month after that](https://www.npmjs.com/package/jquery-ui-dist/v/1.13.2).

We use the latest version (which is 1.13.2) in the [latest v10 and v11](https://github.com/umbraco/Umbraco-CMS/blob/v11/dev/src/Umbraco.Web.UI.Client/package.json#L47) versions of Umbraco.

**Pull request merged**